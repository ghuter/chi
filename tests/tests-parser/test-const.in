a :: 10;
b :: "Hello";
c :: 0.1;
d :: Point {x: 0, y: 0};
