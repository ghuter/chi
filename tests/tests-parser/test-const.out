const SDECL(a, ECSTI(10)) : ?
const SDECL(b, ECSTS(Hello)) : ?
const SDECL(c, ECSTF(1.000000)) : ?
const SDECL(d, ESTRUCT(Point { <x : ECSTI(0)> <y : ECSTI(0)> })) : ?
